viewTask.html

```diff
@@ -222,8 +223,56 @@ <h4>Today</h4>
                 </div>
             </div>
         </div>
+
+        <!-- Create Task Modal Start-->
+        <div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+            <div class="modal-dialog-centered modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <input type="text" class="modal-title modal-title-input shadow-none" placeholder="Title here">
+                        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    </div>
+                    <textarea rows="5" placeholder="Description" class="modal-body desc-input"></textarea>
+                    <div class="modal-footer">
+                        <div class="w-100 date-icon-container">
+                            <input type="date" class="date-input">
+                            <div class="icon-container">
+                                <span class="bi bi-tag fs-5"></span>
+                                <span class="bi bi-flag fs-5"></span>
+                                <span class="bi bi-clock fs-5"></span>
+                            </div>
+                        </div>
+                        <div>
+                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Cancel</button>
+                            <button type="button" class="btn btn-primary">Add Task</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Create Task Modal End -->
     </main>
 
+    <style>
```

Comment:
Create a new file styles.css and move the styles in that file and then add that file in the head.